packages/amplify_flutter/lib/src/config/amplify_config.dart

```diff
@@ -1,47 +1,69 @@
-/*
- * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- * You may not use this file except in compliance with the License.
- * A copy of the License is located at
- *
- *  http://aws.amazon.com/apache2.0
- *
- * or in the "license" file accompanying this file. This file is distributed
- * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing
- * permissions and limitations under the License.
- */
-
-import 'package:amplify_flutter/src/config/analytics/analytics_config.dart';
-import 'package:amplify_flutter/src/config/api/api_config.dart';
-import 'package:amplify_flutter/src/config/auth/auth_config.dart';
-import 'package:amplify_flutter/src/config/storage/storage_config.dart';
+//
```

Comment:
Is there any particular reason we are switching from ` /* comment */` to `/// comment`?  Asking out of curiosity.

Reply:
I mean, it might be. I used Google's tool to add them automatically: https://github.com/google/addlicense. Google's code use `//` from what I've seen.

Reply:
Ok, let's go with it then.